StreamChatUITestsApp/DebugMenu.swift

```diff
@@ -62,7 +62,33 @@ final class DebugMenu {
                                                     message: channelController.channel?.lastActiveMembers.map(\.name).debugDescription,
                                                     actions: []
                                   )
-                        })
+                              }),
+                        .init(title: "Truncate channel w/o message",
+                              style: .default,
+                              handler: { _ in
+                                channelController.truncateChannel { [unowned self] error in
+                                    if let error = error {
+                                        self.presentAlert(in: viewController,
+                                                          title: "Couldn't truncate channel",
+                                                          message: "\(error.localizedDescription)",
+                                                          actions: []
+                                        )
```

Comment:
To keep the code consistent, we should move all the parameters into a new line, to keep things tidier. This for formatting is the old Objc one and we should avoid it 👍 (At least in this codebase for consistency)
```suggestion
                                        self.presentAlert(
                                                 in: viewController,
                                                 title: "Couldn't truncate channel",
                                                 message: "\(error.localizedDescription)",
                                                 actions: []
                                        )
```

Reply:
Makes sense 👍